tests/unit/repositories/ContentReadRepositoryTest.php

```diff
@@ -223,5 +223,148 @@ public function canPaginateResults()
         $this->assertEquals('B title', $result->first()->translations->first()->title);
         $this->assertEquals('A title', $result->last()->translations->first()->title);
     }
+
+    /** @test */
+    public function getManyPublishedShouldHideTranslationsForInactiveRoutes()
```

Comment:
I don't like those long title, could you remove method prefix and simplify them a bit?

Reply:
Yes, ofc.